app/assets/stylesheets/layouts/scaffolds.scss

```diff
@@ -19,7 +19,7 @@ pre {
   font-size: 11px;
 }
 
-a {
+main a {
```

Comment:
Something I noticed during the upgrade -- this selector should not apply in the footer, otherwise we get extremely low contrast when hovering over the links in the lux footer.